src/main/scala/cromwell/engine/finalcall/CopyWorkflowOutputsCall.scala

```diff
@@ -19,7 +18,7 @@ object CopyWorkflowOutputsCall {
   * Final call implementation that copies workflow outputs to a specified destination.
   */
 case class CopyWorkflowOutputsCall(override val workflow: WorkflowDescriptor) extends FinalCall {
-  override def unqualifiedName = CopyWorkflowOutputsCall.name(workflow.name)
+  override def unqualifiedName = CopyWorkflowOutputsCall.UnqualifiedName
```

Comment:
I'd prefer to not need the CopyWorkflowOutputsCall part required here but don't care too much if you'd like to retain it